app/controllers/users_controller.rb

```diff
@@ -0,0 +1,52 @@
+class UsersController < ApplicationController
+
+  ######## SIGN UP #########
+  get '/signup' do
+    if is_logged_in?
+      redirect to '/tweets'
+    else
+      erb :'/users/create_user'
+    end
+  end
+
+  post '/signup' do
+    @user = User.new
```

Comment:
lets use remove the instance variable here and use User.create here and simplify this
```
user = User.create(username: params[:username], email: params[:email], password: params[:password])
if user.save
  login(user.username, user.password)
```